modules/ROOT/pages/customsidebar.adoc

```diff
@@ -6,6 +6,8 @@
 
 {productname} allows developers to create sidebars and add custom UI widgets inside a constrained and easily accessible area of the editor. The sidebar is designed to allow administrators and plugin developers to provide additional tools that can be accessed by {productname} users.
 
+NOTE: Sidebars do not work when {productname} is run in xref:use-tinymce-inline.adoc[inline editing mode].
```

Comment:
This isn't quite accurate... we should be saying it only works in classic/iframe mode as it also won't work for distraction free mode (though technically is a slimmed down inline mode).

This is also inconsistent with the opposite where we have https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/modules/ROOT/partials/misc/admon-inline-only.adoc already. So I'd expect this to largely be the same as that just worded differently for classic/iframe mode 🤷 the linked one can of course be changed, the main thing is consistency.

Reply:
To quote my commit message:

> Edited NOTE admonition in both files to be more explicit about what is and is not supported, and to make both admonitions consistent with each other.